README.md

```diff
@@ -11,84 +11,96 @@ States, which are stored after a certain interval, are used for that.
+**dead_connections_scan_interval: _100000_** — this parameter specifies the time after which the scanning of dead 
+connections is run
 
-**parse_fix: _false_** — to be removed?
+**connection_death_interval: _10800000_** — this parameter specifies the time after which the connection 
+is considered dead
 
-**fix_connection_death_interval: _10800000_** — this parameter specifies the time after which the connection is considered dead.
+**read_state: _true_** — this parameter specifies if the component reads its state when (re)staring in order to 
+resume the processing
 
-**read_state: _true_** — this parameter specifies if the component reads its state when (re)staring in order to resume the processing.
+**write_state: _true_** — this parameter specifies if the component write its state which is used 
+to resume the processing
 
-**write_state: _true_** — this parameter specifies if the component write its state which is used to resume the processing.
+**use_timestamp_from_pcap: _true_** — this parameter specifies if timestamps of the raw messages will be set 
+as they are in PCAPs (true) or as the current time on the server (false)
 
-**use_timestamp_from_pcap: _true_** — this parameter specifies if timestamps of the raw messages will be set as they are in pcaps (true)
-or as the current time on the server (false).
+**sleep_interval: _60_** — number of seconds(?) that the component will wait after finishing processing of all 
+available packets in PCAPs before
+checking if there's more to be processed
 
-**sleep_interval: _60_** — number of seconds(?) that the component will wait after finishing processing of all available packets in pcaps before
-checking if there's more to be processed.
+**number_of_packets_to_successful_restore: _10_** — the number of packets needed to successfully restore a gap 
+in a stream message stream
 
-**number_of_packets_to_successful_restore_fix: _10_** —
+**use_event_publish: _true** — this parameter specifies if the component will publish any events
 
-**use_event_publish: _true** — this parameter specifies if the component will publish any events.
+**disable_cradle_saving: _false_** — this parameter specifies if the component will store the messages to 
+[cradle](https://github.com/th2-net/cradleapi)
 
-**disable_cradle_saving: _false_** — this parameter specifies if the component will store the messages to [cradle](https://github.com/th2-net/cradleapi).
+**disable_connectivity_saving: _false_** — this parameter specifies if the connection event will be stored
 
-**disable_connectivity_saving: _false_** —
+**event_batch_size: _1048576_** — this parameter specifies the size of an event batch (1MB by default)
 
-**event_batch_size: _1048576_** — this parameter specifies the size of an event batch (1MB by default).
+**event_batcher_core_pool_size: _2_** —  this parameter specifies the number of events necessary for sending another 
```

Comment:
```suggestion
**event_batcher_core_pool_size: _100_** —  this parameter specifies the number of events necessary for sending another 
```

Reply:
Why do we call it `pool_size`? In my understanding start sounds like something related to the thread inside the application